api/service.py

```
@@ -9,14 +9,16 @@
 from collections import deque
-from fastapi import FastAPI, HTTPException, status, Depends
-from fastapi.middleware.cors import CORSMiddleware
-from fastapi.security import HTTPBasic, HTTPBasicCredentials
+from fastapi import FastAPI, HTTPException, status, Depends, Request
 from contextlib import asynccontextmanager
 import secrets
 from core.secrets import get_secret, mask_secret
 from pydantic import BaseModel
 
+# Import centralized secrets management
+from core.secrets import get_secret, get_secret_masked
```

Ruff (0.14.10)

Redefinition of unused `get_secret` from line 15

Remove definition: `get_secret`

(F811)